enclave/Enclave.edl

```diff
@@ -22,13 +22,17 @@ enclave {
     from "sgx_stdio.edl" import *;
     from "sgx_backtrace.edl" import *;
     from "sgx_tstdc.edl" import *;
+
     trusted
     {
         /* ECALLs */
         public sgx_status_t ecall_test([in, size=len] const uint8_t* some_string, size_t len);
+        public sgx_status_t exec_wasm_test([in, count=binary_len] const uint8_t* binary, size_t binary_len, [out] int32_t* result_out);
```

Comment:
Sorry, can you please explain again why you used `count=binary_len` instead of `size=binary_len`.

From pg 47 of https://01.org/sites/default/files/documentation/intel_sgx_sdk_developer_reference_for_linux_os_pdf.pdf:
> Buffer Size Calculation
The generalized formula for calculating the buffer size using these attributes:
Total number of bytes = count * size
> * The above formula holds when both count and size/sizefunc are
specified.
> * size can be specified by either size or sizefunc attribute.
> * If count is not specified for the pointer parameter, then it is assumed to
be equal to 1, for example, count=1. Then total number of bytes equals
to size/sizefunc.
> * If size is not specified, then the buffer size is calculated using the
above formula where size is sizeof (element pointed by the pointer).

Reply:
When working with byte-sized types, `count` and `size` will have the same effect. 

The reason I prefer to use `count` is that it is more in line with what I am expressing in the code, since `binary_len` in this case refers to the length of the array, and not the size of the array in bytes (even though in this case it is the same).

If you look at the last bullet point, the formula when using `count` instead of size would be `Total number of bytes = count * sizeof(uint8_t)`

Reply:
Ah I understand, thanks for the explanation! :)